src/NHibernate/Type/TicksType.cs

```diff
@@ -9,46 +7,47 @@
 namespace NHibernate.Type
 {
 	/// <summary>
-	/// Maps a <see cref="System.DateTime" /> Property to an <see cref="DbType.Int64" /> column 
+	/// Maps a <see cref="System.DateTime" /> Property to an <see cref="DbType.Int64" /> column
 	/// that stores the DateTime using the Ticks property.
 	/// </summary>
 	/// <remarks>
-	/// This is the recommended way to "timestamp" a column.  
-	/// The System.DateTime.Ticks is accurate to 100-nanosecond intervals. 
+	/// This is the recommended way to "timestamp" a column, along with <see cref="UtcTicksType" />.
+	/// The System.DateTime.Ticks is accurate to 100-nanosecond intervals.
+	/// This type yields dates with an unspecified <see cref="DateTime.Kind"/>. On writes, it
+	/// does not perform any checks or conversions related to the kind of the date value to persist.
 	/// </remarks>
 	[Serializable]
-	public partial class TicksType : PrimitiveType, IVersionType, ILiteralType
+	public partial class TicksType : AbstractDateTimeType
```

Comment:
This type should have been refactored that way when `AbstractDateTimeType` has been put in place. In order to avoid duplicating the datetime kind handling, I have done it in this PR.

I am right thinking this is not a binary breaking change, provided `AbstractDateTimeType` inherits itself all the removed types and do not have "less virtual" interface implementations?

Reply:
Yes, that's fine